src/atcore.cpp

```diff
@@ -150,8 +151,22 @@ QList<QSerialPortInfo> AtCore::serialPorts() const
 void AtCore::newMessage(const QByteArray &message)
 {
     lastMessage = message;
+    if (message.startsWith(QString::fromLatin1("X:").toLocal8Bit())) {
+        posString = message;
+        posString.resize(posString.indexOf("E"));
+        posString.replace(":", "");
+    }
```

Comment:
I have started to add checks within our normal checkRecvMessage functions i think that is better then having several that way at least all the logic for checking the message is in the same place.. (btw the only way to get that string is to send M114)